python/gui/auto_generated/qgsdoublevalidator.sip.in

```diff
@@ -50,26 +50,38 @@ Constructor for QgsDoubleValidator.
 :param parent: parent object
 %End
 
-    QgsDoubleValidator( double bottom, double top, int decimal, QObject *parent );
+    QgsDoubleValidator( double bottom, double top, int decimals, QObject *parent );
```

Comment:
This is an api break -- we can't change argument names without potentially breaking PyQGIS code which uses named arguments